internal/mesh/internal/controllers/sidecarproxy/builder/testdata/destination/l4-multiple-implicit-destinations-tproxy.golden

```diff
@@ -61,57 +61,6 @@
       }
     },
     "listeners": [
-      {
```

Comment:
These are removed because they are extraneous (one per destination instead of one for all implicit destinations)